CHANGELOG.md

```diff
@@ -4,6 +4,147 @@ All notable changes to the Sorare GraphQL API will be documented in this file. W
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 2023-04-03
+
+Removed the following deprecated fields:
+
+<details>
+<summary>See the list</summary>
+
+- `Account.oldId`: use id
+- `Card.belongsToUser`
+- `Card.canSell`
+- `Card.cardEdition`
+- `Card.cardPrint`: use cardEdition and layout instead
+- `Card.club`: replaced by team
+- `Card.currentSeasonBonus`: replaced by power breakdown
+- `Card.gameForFixture`: use gameForLeague instead
+- `Card.layout`
+- `Card.license`
+- `Card.nextVaultId`: not needed. This info is provided in prepare mutations
+- `Card.owners`: use notContractOwners
+- `Card.playerInjuries`
+- `Card.price`: not relevant
+- `Card.sameClubBonus`: replaced by power breakdown
+- `Card.scoreForPreviousFixture`: use the So5Scores of the player instead
+- `Card.teamPictureUrl`: use pictureUrl on team instead
+- `Card.userOwnerWithRate`: replaced by ownerWithRate
+- `Card.userOwners`: use notContractOwners
+- `Card.userOwnersWithRate`: use notContractOwners
+- `Card.vaultId`: not needed. This info is provided in prepare mutations
+- `CardSubscription.belongsToUser`
+- `CardSubscription.canSell`
+- `CardSubscription.cardEdition`
+- `CardSubscription.cardPrint`: use cardEdition and layout instead
+- `CardSubscription.club`: replaced by team
```

Comment:
I don't really get why it `CardSubscription` here

Reply:
I think because `CardSubscriptionType` extends `CardType`